README.md

```diff
@@ -159,6 +159,19 @@ Mixlib::Install.available_versions("chef", "stable")
 # => ["12.13.3", "12.13.7"]
 ```
 
+### Download an artifact
+Download a specific artifact to a configurable location. All platform options (platform, platform_version, architecture) are required  in order to filter a single artifact.
+```ruby
+# detect platform and download to "/tmp"
+Mixlib::Install.new(product_name: "chefdk", channel: :stable).detect_platform.download_artifact("/tmp")
```

Comment:
In the event the example is cargo culted; `Dir.tmpdir`  is a better non-platform specific path rather than `"/tmp"`.

Reply:
`Dir.tmpdir` is the default value for the parameter in the code.

Reply:
I see what you mean now. I'll update this